lessons/Projects/project-snack-habits/langs/en-us/index.adoc

```diff
@@ -0,0 +1,194 @@
+- *Link the Google Form Responses to a Google Sheet* from the Responses of the form by clicking the button that reads *Link to Sheet*. The sheet will automatically continue to populate the Google Sheet each time the form gets submitted.
+- When your class is done collecting data, 
+  * duplicate the sheet in a new tab 
+  * name the new tab "cleaned" 
+  * remove the timestamp column and the image column
+  * make sure the sharing settings are set to "Anyone with the link can view"
+@ifproglang{pyret}{
+  * copy the url into the @starter-file{snack} to import the data, publish the file and share a link with your students
+}
+@ifproglang{codap}{
+  * import the spreadsheet into CODAP and share a link with your students
+}
+
+We have set the form up with data verification for most questions to minimize the amount of cleaning you will have to do, but we would expect that the one word answers to "Why are you eating this snack?" will require some attention to address: 
+
+- typos
+- inconsistent capitalization
+- words that mean the same thing as each other and should be combined 
+- perhaps there are a number of nonsense words that might make more sense to be replaced by "idk" why I'm snacking?
+
+If you have the time, we encourage you to project the spreadsheet and tackle cleaning the data with your students, asking:
+
+- What inconsistencies do you see in the data?
+- How should we address them?
+
+Ideally, by the time you're done you could make a pie-chart of the "why" column and the breakdown of reasons would be informative. 
+}
+
+@lesson-instruction{
+- Now that we have finished collecting our data, let's see what we can learn about our habits!
+- Open the starter file I shared with you and use it to complete @handout{data-cycle-categorical.adoc}
+}
+
+@teacher{@right{@image{images/pie-chart-salty-sweet.png, 500}} For reference, here's an example of a display from one Bootstrap teacher's class. Consider the kinds of discussions your class might have if the data from your class came back looking like this.}
+
+=== Synthesize
+
+- What did you learn?
+- What new questions do you have?
+
+== Research: U.S. Snacking Habits Data
+
+=== Overview
+
+Students will gather information from studies about US American snacking habits and compare them to the data we've just gathered as a class.
+
+=== Launch
+
+@lesson-instruction{
+Do you think the sample we've made of our snack habits is representative of snacking in the United States?
+}
+
+We don't have to guess! There's plenty of research out there for us to look at. In fact, enough data has been collected about childhood obesity in the United States that soda machines and unhealthy snacks have been pulled from many schools.
+
+=== Investigate
+
+@lesson-instruction{
+- Open your favorite search engine (Google, Brave, DuckDuckGo, etc.) and type +
+"What do Americans eat for snack?”
+- Look for graphs and statistics that relate to the kinds of data we're collecting about our own snacking habits.
```

Comment:
did you actually try entering this prompt into a search engine? In google, I got lots of pictures of snacks, a few charts and articles, and many lists of like "top 10 american snacks: corn dogs, rice krispy treats, etc"

Instructing them to type "charts of american snacking habits" and then go to "images", they'll get heaps of charts. Seems more efficient? I'm thinking of a middle school audience.

Reply:
This whole section seems weirdly brief and sorta underdeveloped. I think I would bump the investigate section to the launch, and then add some discussion questions to help students process what they observe in the graphs that they locate.

Then, I would move the Synthesize section to become the investigate section, and flesh that out much more - e.g., add a worksheet and the option to start adding to the slide deck (or at least a prompt to save worthwhile graphs, stats, and URLs to credit sources).

I would love to add a synthesize question that invites students to consider the data collection process they just underwent *versus* whatever data collection processes were used to create the graphs and charts they found in google., e.g.: How are the sample sizes different? Which charts contain more credible data - the ones they built in Pyret or the ones they found via google search? Why?